rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -308,13 +316,13 @@ static bool encl_build(struct sgx_secs *secs, void *bin, unsigned long bin_size,
 	return true;
 out_map:
 	free(ssa_frame);
-	munmap((void *)secs->base, secs->size);
+	munmap((void *) secs->base, secs->size);
 out_dev_fd:
 	close(dev_fd);
 	return false;
 }
 
-static bool get_file_size(const char *path, off_t *bin_size)
+static bool get_file_size(const char *path, off_t * bin_size)
```

Comment:
Looks like only this place is not expected. Plz fix this kind of indent issue manually.